test/integration/visitor_can_view_items_test.rb

```diff
@@ -38,6 +40,8 @@ class VisitorCanViewItemsTest < ActionDispatch::IntegrationTest
 
     click_link "Coconut Oil"
 
+    assert current_path, '/oil_types/coconut_oil'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.